simplified-accounts-api/src/main/java/org/nypl/simplified/accounts/api/AccountProviderType.kt

```diff
@@ -231,15 +235,19 @@ interface AccountProviderType : Comparable<AccountProviderType> {
       addLink(links, uri, "privacy-policy")
     }
 
-    return AccountProviderDescription(
-      id = this.id,
-      title = this.displayName,
-      updated = this.updated,
-      links = links.toList(),
-      images = imageLinks.toList(),
-      isAutomatic = this.addAutomatically,
-      isProduction = this.isProduction
-    )
+    val accountProviderDescription =
+      AccountProviderDescription(
+        id = id,
+        title = displayName,
+        updated = updated,
+        links = links.toList(),
+        images = imageLinks.toList(),
+        isAutomatic = addAutomatically,
+        isProduction = isProduction
+      )
+
+    check((this.authenticationDocumentURI != null) == (accountProviderDescription.authenticationDocumentURI != null))
```

Comment:
Is there a test we could add or modify to catch this in the future?

Reply:
There should be! By that I mean that I don't think there is right now...